interface/wx/translation.h

```diff
@@ -167,9 +177,10 @@ class wxTranslations
             selected or system-default languages, but is not necessarily an
             error if no translations are needed in the first place.
 
-        @since 3.3.0
+        @since 3.2.5
```

Comment:
Sorry, I should have commented on this. I know (the 3.2 branch documents it correctly), but I figured it's a useful white lie. It is *technically* true and because the pre-3.2.5 version was broken by design, and x.y.(z+1) updates are not problematic, it IMHO makes sense to just document the corrected version...